Tests/fixtures/console.yml

```diff
@@ -379,7 +379,10 @@
   browser_family: Chrome
 -
   user_agent: Mozilla/5.0 (PlayStation; PlayStation 5/2.26) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Safari/605.1.15
-  os: [ ]
+  os:
+    name: PlayStation
+    version: "5"
+    platform: ""
```

Comment:
It seems that each gen of consoles has its own versions software
https://www.reddit.com/r/playstation/wiki/guides/system-software/ps5

https://www.reddit.com/r/playstation/wiki/guides/system-software/ps4
https://en.wikipedia.org/wiki/PlayStation_4_system_software

here we need to think about how to specify the version correctly

`Mozilla/5.0 (PlayStation; PlayStation 5/2.26) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Safari/605.1.15`
versions: 5/2.26 or 5.2.26 for PS5

`Mozilla/5.0 (PlayStation; PlayStation 4/9.03) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0 Safari/605.1.15`
versions: 4/9.03 or 4.9.03 for PS4
`Mozilla/5.0 (PlayStation 4 3.11) AppleWebKit/537.73 (KHTML, like Gecko)`
versions: 4/3.11 or 4.3.11  for PS4

@sgiehl What version format is acceptable to us?

Reply:
As we don't use `/` in versions yet, maybe using a `.` instead would be better.

Reply:
@liviuconcioiu 
set full version

`Mozilla/5.0 (PlayStation; PlayStation 5/2.26) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.0 Safari/605.1.15`
``` yml
regex: 'PlayStation (\d)(?:[/ ](\d\.))?'
name: 'PlayStation'
vesion: '$1.$2'    // 5.2.26
```